Library/Homebrew/cmd/test-bot.rb

```diff
@@ -155,9 +154,12 @@ def run
           working_dir.cd { exec(*@command) }
         end
         write.close
-        while line = read.gets
-          puts line if verbose
-          @output += line
+        while buf = read.read(32)
+          if verbose
+            print buf
+            $stdout.flush
+          end
+          @output << buf
```

Comment:
We could make it lower just for Travis. If we're printing dots based on time output it seems to make sense to ensure that we output every dot otherwise they will e.g. just appear in groups of `16`.